### PR TITLE
Facebook feed

### PR DESCRIPTION
**Pull request by dani5447**
feedupdate

### CODE REVIEW

**Comment by JRJurman** [comment]
make filter for picture only stuff

File: app/js/index.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import Navbar from './components/navbar';
+import StatusUpdate from './components/fb-status-update';
```
Comment (JRJurman) [inline app/js/index.js]: do we need this?

File: app/js/components/fb-feed.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import Story from './fb-story';
+
+export default class Feed extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {feed:[]}
+  }
+
+  componentDidMount() {
+    var that = this;
```
Comment (bbesmanoff) [inline app/js/components/fb-feed.js:12]: you may be able to get away with using all arrow functions, but this works as well

**Comment by JRJurman** [comment]
:+1:

**Pull request merged**